doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,247 @@
+# Getting Started
+
+Welcome to libp2p! This guide will walk you through setting up a fully functional libp2p node 🚀
+
+- [Getting Started](#getting-started)
+  - [Install](#install)
+  - [Configuring libp2p](#configuring-libp2p)
+    - [Basic setup](#basic-setup)
+      - [Transports](#transports)
+      - [Connection Encryption](#connection-encryption)
+      - [Multiplexing](#multiplexing)
+      - [Running Libp2p](#running-libp2p)
+    - [Custom setup](#custom-setup)
+      - [Peer Discovery](#peer-discovery)
+      - [Pubsub](#pubsub)
+  - [What is next](#what-is-next)
+
+## Install
+
+The first step is to install libp2p in your project:
+
+```sh
+npm install libp2p
+```
+
+## Configuring libp2p
+
+If you're new to libp2p, we recommend configuring your node in stages, as this can make troubleshooting configuration issues much easier. In this guide, we'll do just that. If you're more experienced with libp2p, you may wish to jump to the [Configuration readme](./CONFIGURATION.md).
+
+### Basic setup
+
+Now that we have libp2p installed, let's configure the minimum needed to get your node running. The only modules libp2p requires are a [**Transport**][transport] and [**Crypto**][crypto] module. However, we recommend that a basic setup should also have a [**Stream Multiplexer**](streamMuxer) configured, which we will explain shortly. Let's start by setting up a Transport.
```

Comment:
Is the muxer really optional?

Reply:
Yes, you should be able to run an unmuxed connection. The primary reason would be to support applications that don't need/want it, such as lower energy/power devices, although this may not be a valid use case for js-libp2p.